index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>Rock, Paper, Scissors</title>
+</head>
+
+<body id="body">
+<header id="header">
+    <h1>Rock, Paper, Scissors!</h1>
+</header>
+
+<section id= "rules">
+<h3><bold><span id= "rule-heading">The Rules</span></bold></h3>
```

Comment:
Looks like you're suing some tags to style your app. Remember that styling should be done in CSS not the HTML.